python3/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM socrata/base
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python3 python3-dev python3-pip
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.4 1
```

Comment:
That's exactly what I was trying to do there, yeah. What do you mean that they're not alternatives? I found that tip here: https://linuxconfig.org/how-to-change-from-default-to-alternative-python-version-on-debian-linux (and other similar tips elsewhere).